### ISSUE TITLE
Getting get_data_slice_zero(): incompatible function arguments when changing viewer to Datagrid plugin

### ISSUE THREAD

**Issue by Yuri2b** [finos/perspective#1115]
### Steps to Reproduce:
 1. Open `.arrow` file by `pyarrow RecordBatchFileReader`
 2. Create `perspective Table` from the file
 3. Host table by `PerspectiveManager`
 4. Call `perspectiveViewer.load(websocket.open_table("table_name"))` in browser
 5. Change visualization plugin to `Datagrid`

### Expected Result:
Changing perspective-viewer plugin to `Datagrid`

### Actual Result:
I get this error from server side: 
```
get_data_slice_zero(): incompatible function arguments.
The following argument types are supported:    1. (arg0: perspective.table.libbinding.View_ctx0, arg1: int, arg2: int, arg3: int, arg4: int) -> perspective.table.libbinding.t_data_slice_ctx0Invoked
with: <perspective.table.libbinding.View_ctx0 object at 0x000002B5805A3F70>, 0, 40.78947368421053, 0, 1
```

Another plugins, including `Hypergrid`, work as expected

On node.js server with the same data file all works as expected

Client's message from websocket pipe that cause the error:

```json
{"cmd":"view_method","name":"0.720999804799138","method":"to_columns","args":[{"start_col":0,"end_col":2,"start_row":0,"end_row":40.473684210526315,"id":false}],"subscribe":false,"id":248}
```

and answer of server is:
```json
{"id": 248, "error": "get_data_slice_zero(): incompatible function arguments. The following argument types are supported:\n    1. (arg0: perspective.table.libbinding.View_ctx0, arg1: int, arg2: int, arg3: int, arg4: int) -> perspective.table.libbinding.t_data_slice_ctx0\n\nInvoked with: <perspective.table.libbinding.View_ctx0 object at 0x000001DF8F092130>, 0, 40.473684210526315, 0, 2"}
```

Should `end_row` in client's message be integer rather than float?

### Environment:
Windows  10.0.18363
python 3.7.8
perspective-python 0.5.1

Schema of my arrow file:
```
{'index': <class 'int'>, 'gbuName': <class 'str'>, 'periodName': <class 'str'>, 'ind': <class 'str'>, 'indName': <class 'str'>, 'values': <class 'float'>, 'Y': <class 'str'>}
```

**Comment by sc1f** [comment]
Thanks for the detailed report! This issue is fixed in #1112, as well as in the new version of [regular-table](https://github.com/jpmorganchase/regular-table/) which is being implemented as the newest version of datagrid. Will keep this issue open until those fixes are merged.

**Comment by texodus** [comment]
Fix released in v0.5.2
